src/nvim/CMakeLists.txt

```diff
@@ -65,6 +65,10 @@ foreach(sfile ${NEOVIM_SOURCES})
   if(${f} MATCHES "^(regexp_nfa.c)$")
```

Comment:
regexp_nfa.c is included in regexp.c so that is why we don't put it in the sources var, it still needs to generate headers though.